core/src/services/onedrive/graph_model.rs

```diff
@@ -0,0 +1,204 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use serde::{Deserialize, Serialize};
+use serde_json::Result;
+use std::collections::HashMap;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub(crate) struct GraphApiOnedriveResponse {
+    #[serde(rename = "@odata.context")]
+    odata_context: String,
+
+    #[serde(rename = "@odata.count")]
+    pub(crate) odata_count: usize,
+
+    pub(crate) value: Vec<OneDriveItem>,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub(crate) struct OneDriveItem {
+    #[serde(rename = "createdDateTime")]
+    created_date_time: String,
+    #[serde(rename = "eTag")]
+    e_tag: String,
+    id: String,
+    #[serde(rename = "lastModifiedDateTime")]
+    last_modified_date_time: String,
+    pub(crate) name: String,
+
+    size: usize,
+    #[serde(rename = "webUrl")]
+    web_url: String,
+    #[serde(rename = "parentReference")]
+    pub(crate) parent_reference: ParentReference,
+    #[serde(rename = "fileSystemInfo")]
+    file_system_info: FileSystemInfo,
```

Comment:
@xuanwo Shall I remove all unused fields of these structs?

Reply:
Yes.